project/Rat.scala

```diff
@@ -22,8 +22,13 @@ object Rat {
   lazy val excludes = Seq(
     // git files
     file(".git"),
-    // json files -- these ones do not support comments
-    file("snippets/dfdl.json"),
+    /**
+      * Can't add license headers in JSON files.
+      * Adding a license attribute breaks things in some of these fiels as well.
```

Comment:
@rt320 would you be able to comment on this? I am not sure on `language/dfdl.json`.
`language/syntaxes/dfdl.tmLanguage.json` seems to be based off something however since I noticed this in the file
```json
"information_for_contributors": [
		"This file has been converted from https://github.com/atom/language-xml/blob/master/grammars/xml.cson",
		"If you want to provide a fix or improvement, please create a pull request against the original repository.",
		"Once accepted there, we are happy to receive an update request."
	]
```
but I would say its possible `language/syntax.json` was gotten from somewhere